src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java

```diff
@@ -40,6 +43,34 @@ private static void setupPresetTests(Application application) {
 		SelectJaneOptionListener selectJaneOptionListener = new SelectJaneOptionListener(
 				DriverFeature.getDriverManager());
 		application.addTest("Figure Jane", selectJaneOptionListener);
+
+
+		ActionListener rectangle = new ActionListener()
+		{
+			public void actionPerformed(ActionEvent e) {
+				ShapesFactories.drawRectangle(0,0, 50, 25).execute(DriverFeature.getDriverManager().getCurrentDriver());
+			}
+		};
```

Comment:
It is good to keep a convention used in the original code (concrete class instead of anonymous class)